src/develop/ui/patterns/mobile/carousel.md

```diff
@@ -87,44 +82,42 @@ You can use CSS to customize the look and feel of the dots.
 
 ![](images/Carousel_before_after_2.png)
 
-### Using the Carousel Inside Columns
+### Using the Carousel Pattern inside the Columns Pattern
 
-If you want to use a Carousel inside any [Columns pattern](<columns.md>) , you must fix the columns’ overflow, by adding this style
-to the CSS:
-
-    
+To use the Carousel Pattern inside any Columns Pattern , you must fix the columns’ overflow, by adding the following style to the CSS:
+   
     
     .col {
         overflow: hidden;
     }
     
 
-### Getting the CurrentPosition of a Carousel Item
+### Getting the current position of a Carousel Item
 
-  1. Create an **Integer** local variable (ex: Position) on the screen. 
-  2. In the **OnItemChange** event of the block, create a new client action for the handler (CarouselCurrentPosition).   
-a. The action receives a variable **Index** , which is the event that
+  1. Create an **Integer** local variable (for example, Position) on the screen. 
+  1. In the **OnItemChange** event of the block, create a new client action for the handler (CarouselCurrentPosition).   
+    a. The action receives a variable **Index** , which is the event that
 indicates the current position in the Carousel.
 
-  3. Drag an **Assign** node and set the Position (your local variable) to Index. 
+  1. Drag an **Assign** node and set the Position (your local variable) to Index. 
 
 ![](images/Carousel_current_position.png)
 
-### Creating an Onboarding with the Carousel Pattern
+### Creating an onboarding screen using the Carousel Pattern
```

Comment:
This should probably be the prime example, no? Right where we currently have "Customizing the dots". @pacferreira